### PR TITLE
Refactor scripts

### PR DESCRIPTION
**Pull request by systimotic**
Refactoring these scripts so they used the same `readDir` had been on my todo list for a while.
What this PR does:
- Refactor `readDir` into a util
- Refactor `pagesDir` into a util
- Move both scripts into a `bin` folder, and move the `utils` folder to the root
- Refactor the `info` logging into a util, so it can easily be reused
- Update the build script to run on Windows too, because I was getting tired of having to change it every single time I wanted to test something in the build.
- Add a few more things to the `formatting` array
- Commit the output of `normaliseArticles`

### CODE REVIEW

**Comment by Bouncey** [comment]
Merge Conflict :cry:

File: utils/readDir.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const fse = require('fs-extra');
+
+const { isAFileRE, shouldBeIgnoredRE } = require('./regEx.js');
+
+const excludedDirs = [
```
Comment (Bouncey) [inline utils/readDir.js]: Do we still need this now that we have gotten rid of `src/pages/search/`?

Comment (systimotic) [reply utils/readDir.js]: Nope! Meant to talk about that in the PR that removed it, but forgot, sorry.

**Comment by systimotic** [comment]
@Bouncey Updated!

**Comment by Bouncey** [comment]
## LGTM

Thanks @systimotic 

:sparkles: :sparkles: :sparkles: :sparkles:

**Pull request merged**